CorsixTH/Src/iso_fs.cpp

```diff
@@ -423,36 +423,35 @@ bool iso_filesystem::file_metadata_less(const file_metadata& lhs,
 }
 
 int iso_filesystem::find_hosp_directory(const uint8_t* pDirEnt,
-                                        int iDirEntsSize, int iLevel) {
+                                        const uint32_t dirEntsSize, int level) {
   // Sanity check
   // Apart from at the root level, directory record arrays must take up whole
   // sectors, whose sizes are powers of two and at least 2048.
   // The formal limit on directory depth is 8, so hitting 16 is insane.
-  if ((iLevel != 0 && (iDirEntsSize & (min_sector_size - 1)) != 0) ||
-      iLevel > max_directory_depth)
+  if ((level != 0 && (dirEntsSize & (min_sector_size - 1)) != 0) ||
+      level > max_directory_depth)
     return 0;
 
   std::unique_ptr<uint8_t[]> pBuffer(nullptr);
   uint32_t iBufferSize = 0;
-  iso_directory_iterator dir_iter(pDirEnt, pDirEnt + iDirEntsSize);
-  iso_directory_iterator end_iter(pDirEnt + iDirEntsSize,
-                                  pDirEnt + iDirEntsSize);
+  iso_directory_iterator dir_iter(pDirEnt, pDirEnt + dirEntsSize);
+  iso_directory_iterator end_iter(pDirEnt + dirEntsSize, pDirEnt + dirEntsSize);
   for (; dir_iter != end_iter; ++dir_iter) {
     const iso_file_entry& ent = *dir_iter;
     if (ent.flags & def_directory) {
       // The names "\x00" and "\x01" are used for the current directory
       // the parent directory respectively. We only want to visit these
```

Comment:
```suggestion
      // The names "\x00" and "\x01" are used for the current directory and
      // the parent directory respectively. We only want to visit these
```
Probably fits as new line 438 is longer.